functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/MavenExtensionShadedJarRegisterer.java

```diff
@@ -0,0 +1,65 @@
+package org.arquillian.smart.testing.ftest.testbed.project;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Optional;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+
+class MavenExtensionShadedJarRegisterer {
+
+    MavenExtensionShadedJarRegisterer() {
+    }
```

Comment:
I can see there is no another constructor of `MavenExtensionShadedJarRegisterer`. So no need to declare this